common/app/views/fragments/analytics/google.scala.html

```diff
@@ -103,9 +103,14 @@
         @for(sponsorLogos <- listSponsorLogosOnPage(page)) {
           ga('@{trackerName}.set', 'dimension31', '@Html(sponsorLogos.mkString("|"))');
         }
+
         @for(brandingType <- Commercial.brandingType(page)) {
           ga('@{trackerName}.set', 'dimension42', '@brandingType.name');
         }
+
+        if(document.referrer.includes('oauth') && document.referrer.includes('facebook')) {
```

Comment:
I also thought about some alternative checks, but the best I could come up with was checking for the `GU_U` cookie having an expiry date of 3 months (give or take a few seconds).  That feels worse than what we've gone for though.